src/main/java/com/google/ehub/servlets/ItemPageServlet.java

```diff
@@ -42,14 +44,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       Gson gson = new Gson();
       response.setContentType("application/json");
       response.getWriter().println(gson.toJson(comments));
+      
  }
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long itemId = Long.parseLong(request.getParameter("itemId"));
+    Optional<EntertainmentItem> optionalItem = EntertainmentItemDatastore.getInstance().queryItem(itemId);
+    EntertainmentItem item = optionalItem.get();
+    System.out.println("Item:" + item);
     String message = request.getParameter("text-input");
     long timestamp = System.currentTimeMillis();
     CommentDataManager comments = new CommentDataManager();
-    comments.addItemComment(message,timestamp);
+    comments.addItemComment(itemId,message,timestamp);
```

Comment:
Add spaces after the commas for each argument in the function call. eg. ```addItemComment(itemId, message, timestamp)``` or run prettier/clang-format to make sure the formatting is correct.